src/main/resources/db/migration/V1.21__Create_Legacy_Registrations_Table.sql

```diff
@@ -4,5 +4,6 @@ CREATE TABLE IF NOT EXISTS legacy_registrations (
     hex_id text NOT NULL,
     owner_email text,
     created_date timestamp NOT NULL,
-    last_modified timestamp NOT NULL
+    last_modified timestamp NOT NULL,
+    status text NOT NULL
```

Comment:
@stuart-madetech Flyway gets very unhappy once it has applied a migration script if you change the name of it or change the content within the file.

What you will have to do is leave this file as is and create a new file: `V1.22__Add_Status_Column_To_Legacy_Records_Table.sql` and add the status column

Reply:
Renamed existing file and added new for additional column